configs/eslint-common.json

```diff
@@ -58,7 +57,7 @@
     },
     "overrides": [
         {
-            "files": ["*.ts"],
+            "files": ["*.ts", "*.tsx"],
```

Comment:
Why this change?

Reply:
Right, good catch. We agreed to turn it off only for TS files (RD-2112) my mistake. Reverted that change and updated https://github.com/cloudify-cosmo/cloudify-stage/pull/1796 accordingly. Other PRs not affected.